.rhcicd/clowdapp.yaml

```diff
@@ -18,6 +18,37 @@ objects:
     - policies-ui-backend
     database:
       sharedDbAppName: policies-ui-backend
+    jobs:
+    - name: org-id-populator
```

Comment:
The `policies-ui-backend` app is supposed to own the `policies-ui-backend` DB and it should also take care of migrating its data.

Shouldn't this job by in `policies-ui-backend`?